### PR TITLE
Add venue integration tests

### PR DESCRIPTION
**Pull request by xstrengthofonex**
- Added venue integration tests.
- Added a guard to the delete venue method
- Simplified the update venue method
- Removed a redundant list from the member tests

### CODE REVIEW

File: src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java [hunk 1]
```diff
@@ -64,9 +61,6 @@ private Member createAnotherMember() {
     public void findsAllMembers() throws Exception {
         Member anotherMember = createAnotherMember();
         memberRepository.saveAndFlush(anotherMember);
-        List<Member> members = new ArrayList<>();
-        members.add(aMember);
-        members.add(anotherMember);
```
Comment (TheBeege) [inline src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java:69]: I should have caught this during last review. I'll change my review style. I'll also look into adding a linter

**Review by TheBeege — APPROVED** [review]
Looks good! Sorry it took  me so long to get to it

File: src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java (hunk 1 above)
Comment (xstrengthofonex) [reply src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java:69]: Originally, I was using VS code, which doesn't have many warnings for redundant code, but I switched back to IntelliJ and found a whole bunch of stuff.

**Comment by xstrengthofonex** [comment]
> Looks good! Sorry it took  me so long to get to it

No problem. There's no rush, really. I'll continue to add more tests when I have time. Once there is some coverage, it will be much easier to add new features.

**Pull request merged**